coalib/bearlib/abstractions/Linter.py

```diff
@@ -711,6 +714,8 @@ class GlobalLinterMeta(type(LinterBase), type(GlobalBear)):
 
     class GlobalLinterBase(LinterBase, GlobalBear, metaclass=GlobalLinterMeta):
 
+        LinterParent.register(LinterBase)
```

Comment:
Why not putting this into `LinterBase`? it seems that every class will inherit it